### PR TITLE
Added 1.2.0 version SChinese announcement

### PR DESCRIPTION
**Pull request by CrewCyan**

### CODE REVIEW

**Comment by NikoCat233** [comment]
Dont merge it until cyan fix his errors

File: Patches/AnnouncementPatch.cs [hunk 1]
```diff
@@ -502,6 +502,28 @@ public static void Init()
...
+                //TOHE v1.2.0
+                var news = new ModNews
+                {
+                    Number = 80004,
+                    Title = "Town of Host: Enhanced v1.2.0",
+                    SubTitle = "★★ 支持The Fungle ! ★★",
+                    ShortTitle = "TOH: Enhanced v1.2.0",
+                    BeforeNumber = 80003,
+                    Text = "<size=150%>欢迎来到TOH: Enhanced v1.2.0!</size>\n" +
+                    "\n<b>【基于】</b>\n - 基于TOH: Enhanced v1.1.1\r\n" +
```
Comment (LezaiYa) [inline Patches/AnnouncementPatch.cs:515]: 应该是”【基于官方版本】“（来自咔皮呆翻译）

**Comment by Tommy-XL** [comment]
Could this be merged or it's in progress?

**Comment by NikoCat233** [comment]
Its done, merge it

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**